src/components/Modal/Modal.module.css

```diff
@@ -186,6 +186,11 @@
   height: 100% !important;
   max-width: 100%;
 
+  > div {
```

Comment:
Can we give this div a classname (or target a more specific classname)

previous implementation was fragile and was broken with the antd update; this fix sound as fragile as the previous implementation to me :(

Reply:
You are right, this implementation is fragile. 
Fortunately doing like [here](https://github.com/mia-platform/design-system/pull/593/commits/8abb59a62080dfda99e21de3e7fdf212faed3e28#diff-e7cfca382e4c844d9e6190d694684e0d98ff1f4f03ee5faee86d283036e87af1R191) fixes the issue, and gives a cleaner solution as well